### ISSUE TITLE
Retro68 build fails

### ISSUE THREAD

**Issue by nehalem501** [autc04/Retro68#111]
Building using Multiversal Interfaces works, however when I tried to do a new build of the toolchain using the Apple Universal Interfaces, it fails with the following message:
```
[ 10%] Linking CXX executable ConsoleTest.xcoff
/home/tomek/Retro68-build/toolchain/lib/gcc/powerpc-apple-macos/9.1.0/../../../../powerpc-apple-macos/bin/ld: cannot find -lInterfaceLib
collect2: error: ld returned 1 exit status
Console/CMakeFiles/ConsoleTest.dir/build.make:84: recipe for target 'Console/ConsoleTest.xcoff' failed
make[2]: *** [Console/ConsoleTest.xcoff] Error 1
CMakeFiles/Makefile2:222: recipe for target 'Console/CMakeFiles/ConsoleTest.dir/all' failed
make[1]: *** [Console/CMakeFiles/ConsoleTest.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
```
The Apple Universal Interfaces were downloaded from https://www.macintoshrepository.org/545-macintosh-programmer-s-workshop-mpw-3-x (Last download on the page: "InterfacesAndLibraries.zip (8.62 MiB / 9.04 MB)")

My system is a x86_64 Ubuntu 18.04 machine with GCC 7.5.0

I attached the full log of the build [log.txt](https://github.com/autc04/Retro68/files/5070361/log.txt)

**Comment by autc04** [comment]
Okay, at some point I broke the check for the resource forks of the library files. Should now be fixed with 5f4733ea5fe41711667a25614f0ac4dcc48df9cc, please try again.

Explanation:
Unless you do some manual rearranging of the files that are stored in `__MACOSX__` in the zip archive, Retro68's tools won't find the resource forks of the shared library files. As all we need from the shared library files are a list of function names, Retro68 can fall back to its own lists without losing functionality and without violating copyright law. So the installation script is supposed to check if it can read the resource forks; if yes, it converts them, if not, it uses its own copy. The bug affected the check, so the fallback version was never used.

**Comment by nehalem501** [comment]
Works now, Thanks !
